substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -751,6 +759,28 @@ pub mod pallet {
 	#[pallet::storage]
 	pub(crate) type ChillThreshold<T: Config> = StorageValue<_, Percent, OptionQuery>;
 
+	/// Voter snapshot progress status.
+	///
+	/// If the status is `Ongoing`, it keeps a cursor of the last voter retrieved to proceed when
+	/// creating the next snapshot page.
+	#[pallet::storage]
+	pub(crate) type VoterSnapshotStatus<T: Config> =
```

Comment:
```suggestion
	pub type VoterSnapshotStatus<T: Config> =
```
🙏 otherwise this will be a pain in the Asset Hub migration.

Reply:
ack, will make all public for ya